src/core/src/FabricPrepareRenderResources.cpp

```diff
@@ -80,6 +80,28 @@ std::vector<const CesiumGltf::ImageCesium*> getPropertyTextureImages(
     return MetadataUtil::getPropertyTextureImages(model, primitive);
 }
 
+uint64_t getPropertyTableTextureCount(
+    const FabricMesh& fabricMesh,
+    const CesiumGltf::Model& model,
+    const CesiumGltf::MeshPrimitive& primitive) {
+    if (fabricMesh.material == nullptr) {
+        return {};
```

Comment:
NIT: probably less ambiguous to just return `0`, since the return value is a `uint64_t`.

Reply:
Thanks for catching this, it wasn't intentional, just a copy paste mistake.

Reply:
https://github.com/CesiumGS/cesium-omniverse/pull/573/commits/6e74948cd616e2edeaeee40b147f99c686b72f66